README.md

```diff
@@ -27,8 +27,8 @@ Other available solutions are deficient:
 
 * WINE implements most of `Wintrust`, but is a massive (and arguably non-native) dependency
 for a single task.
-* [`osslsigncode`](https://sourceforge.net/projects/osslsigncode/) can add signatures and check
-timestamps, but is long-abandoned and CLI-focused.
+* [`osslsigncode`](https://github.com/mtrojnar/osslsigncode) can add signatures and check
```

Comment:
Hmm -- this change is already on `master`. Is your local branch stale?

Reply:
Now I wonder.

Reply:
Hmm, no it's not on `master` in your public repo at least. [See here](https://github.com/trailofbits/uthenticode/blame/master/README.md#L30).

Reply:
The README on your public master still has the old link. The fix from November only addressed the one at the bottom of the README.

Reply:
Whoops, my mistake. Thanks for updating this one as wlel.